ui/Screen/Settings.svelte

```diff
@@ -168,6 +169,20 @@
       </div>
     </section>
 
+    <section>
+      <header>
+        <h3>Peer ID</h3>
+      </header>
+      <div class="section-item">
+        <div class="info">
+          <p>Share your peerId with others so they can add you as a remote</p>
```

Comment:
```suggestion
          <p>Share your peerID with others so they can add you as a remote</p>
```

To be consistent with the welcome screen

Reply:
Should is still be called `peerID`? Or `Peer ID`? What's the glossary say?